src/java.desktop/share/classes/com/sun/java/swing/plaf/gtk/XColors.java

```diff
@@ -25,805 +25,531 @@
 
 package com.sun.java.swing.plaf.gtk;
 
-import java.awt.Color;
-import java.util.Arrays;
 import javax.swing.plaf.ColorUIResource;
 
 /**
- * @author  Shannon Hickey
+ * Utility class to convert color names to colors.
+ *
+ * @author Shannon Hickey
+ * @author SWinxy
  */
```

Comment:
You aren't the author, you are changing the file, and if we go anywhere near @author tags it is to remove them, not add them. Plus "Swinxy" is not very professional

Reply:
Done. I wasn't aware of the intent to get rid of the `@author` tags, but it's very valid. They've been removed.